### PR TITLE
[Hotfix] Fix DEFAULT TABLE problem

### PR DESCRIPTION
**Pull request by EricJoy2048**
### Purpose of this pull request

1. Add some log
2. When there is no `table_path` or `database` config on the source side, will use `default` as the database in tableschema.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  3. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  4. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
image

**Comment by hailin0** [comment]
Shall we add test case?

**Comment by EricJoy2048** [comment]
> Shall we add test case?

I do not know how to test this case, only from the log can know the code right.

File: seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/CatalogTableUtil.java [hunk 1]
```diff
@@ -115,14 +115,14 @@ public static List<CatalogTable> getCatalogTables(
...
-                            long startTime = System.currentTimeMillis();
                             try (Catalog catalog = c) {
+                                long startTime = System.currentTimeMillis();
                                 catalog.open();
                                 List<CatalogTable> catalogTables =
                                         catalog.getTables(readonlyConfig);
                                 log.info(
                                         String.format(
-                                                "Get catalog tables, cost time: %d",
+                                                "Get catalog tables, cost time: %d/ms",
```
Comment (Hisoka-X) [inline seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/CatalogTableUtil.java]:
```suggestion
                                                "Get catalog tables, cost time: %d ms",
```

File: pom.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@
         <groupId>org.apache</groupId>
         <artifactId>apache</artifactId>
         <version>31</version>
+        <relativePath />
```
Comment (Hisoka-X) [inline pom.xml:24]:
```suggestion
```

Comment (EricJoy2048) [reply pom.xml:24]: `<relativePath />` is to fix Maven reload error in IDEA.

Comment (Hisoka-X) [reply pom.xml:24]: got it

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**